opencog/atoms/execution/LibraryManager.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * opencog/atoms/execution/LibraryManager.h
+ *
+ * Copyright (C) 2018 OpenCog Foundation
+ * All Rights Reserved
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License v3 as
+ * published by the Free Software Foundation and including the exceptions
+ * at http://opencog.org/wiki/Licenses
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program; if not, write to:
+ * Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef _OPENCOG_LIBRARAY_MANAGER_H
+#define _OPENCOG_LIBRARAY_MANAGER_H
+
+#include <dlfcn.h>
+#include <opencog/atomspace/AtomSpace.h>
+
+class LibraryManager
+{
+private:
+	static std::unordered_map<std::string, void*> _librarys;
+	static std::unordered_map<std::string, void*> _functions;
+public:
+	static void* getFunc(std::string libName,std::string funcName);
```

Comment:
It is worth adding documentation to these methods.

Reply:
LibraryManager was just moved from one file to another, and the newly added interface functions `setLocalSchema` and `setLocalPredicate` have comments regarding their usage. So, again, this is the problem of the original code base - not of this pull request.